pentagon/migration/migrations/migration_2_7_1.py

```diff
@@ -0,0 +1,112 @@
+from copy import deepcopy
+
+
+from pentagon import migration
+from pentagon.migration import *
```

Comment:
Are both these imports needed or could one be summarized? Feels like we're importing the same set of information twice just with different scope contexts.

Reply:
Good point. If there was a reason, I don't recall what it is was. It should be usable with just the import on line 4